pilot/lib/pilot/options.rb

```diff
@@ -19,6 +19,15 @@ def self.available_options
                                      description: "The bundle identifier of the app to upload or manage testers (optional)",
                                      optional: true,
                                      default_value: ENV["TESTFLIGHT_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier)),
+        FastlaneCore::ConfigItem.new(key: :app_platform,
+                                     short_option: "-m",
+                                     env_name: "PILOT_PLATFORM",
+                                     description: "The platform to use (optional)",
+                                     optional: true,
+                                     default_value: "ios",
```

Comment:
It might make sense to change this back to a `nil` default, because as-is it will break `pilot list` which was previously working fine on tvOS-only apps.